setup.cfg

```diff
@@ -54,6 +54,7 @@ tests =
     %(test)s
 dev = 
     %(test)s
+    black
```

Comment:
i've just double checked that `pre commit` will install `black` during the first commit, but it's fine if you want to add it here